---
layout: archive
title: ""
permalink: /publications/
author_profile: true
---

1. Jiang L, Wang M, Lin S, Jian R, Li X, Chan J, Dong G, **Fang H**, Robinson A, GTEx Consortium and Snyder M (2020). [A Quantitative Proteome Map of the Human Body](https://pubmed.ncbi.nlm.nih.gov/32916130/). Cell, 183:1-15.
1. **Fang H**, Hui Q, Lynch J, Honerlaw J, Assimes TL, Huang J, Vujkovic M, Damrauer SM, Pyarajan S, Gaziano JM, DuVall SL, O’Donnell CJ, Cho K, Chang K-M, Wilson PWF, Tsao PS, the VA Million Veteran Program, Sun YV and Tang H (2019). [Harmonizing Genetic Ancestry and Self-identified Race/Ethnicity in Genome-wide Association Studies](https://pubmed.ncbi.nlm.nih.gov/31564439/). The American Journal of Human Genetics, 105: 763-772.
1. Bick AG, Akwo E, Robinson-Cohen C, Lee K, Lynch J, Assimes TL, DuVall S, Edwards T, **Fang H**, Freiberg SM, Giri A, Huffman JE, Huang J, Hull L, Kember RL, Klarin D, Lee JS, Levin M, Miller DR, Natarajan P, Saleheen D, Shao Q, Sun YV, Tang H, Wilson O, Chang K-M, Cho K, Concato J, Gaziano JM, Kathiresan S, O’Donnell CJ, Rader DJ, Tsao PS, Wilson PW, Hung AM, Damrauer SM and the VA Million Veteran Program (2019). [Association of APOL1 Risk Alleles with Cardiovascular Disease in African Americans in the Million Veteran Program](https://pubmed.ncbi.nlm.nih.gov/31337231/). Circulation, 140: 1031-1040.
1. Wang Y, **Fang H**, Yang D, Zhao H and Deng M (2019). [Network clustering analysis using mixture exponential-family random graph models and its application in genetic interaction data](https://pubmed.ncbi.nlm.nih.gov/28858811/). IEEE/ACM Transactions on Computational Biology and Bioinformatics, 16: 1743-1752.
1. Xie W, Luo H, Murugapiran SK, Dodsworth JA, Chen S, Sun Y, Hedlund BP, Wang P, **Fang H**, Deng M and Zhang CL (2018). [Localized high abundance of Marine Group II archaea in the subtropical Pearl River Estuary: implications for their niche adaptation](https://pubmed.ncbi.nlm.nih.gov/29235710/). Environmental Microbiology, 20: 734-754.
1. Ding J, Chen X, **Fang H** and Liu Y (2018). [Case-cohort design for accelerated hazards model](https://www.intlpress.com/site/pub/pages/journals/items/sii/content/vols/0011/0004/a010/index.php). Statistics and Its Interface, 11: 657-668.
1. Coram MA, **Fang H**, Candille SI, Assimes TL and Tang H (2017). [Leveraging Multi-ethnic Evidence for Risk Assessment of Quantitative Traits in Minority Populations](https://pubmed.ncbi.nlm.nih.gov/28757202/). The American Journal of Human Genetics, 101: 218-226.
1. **Fang H**, Huang C, Zhao H and Deng M (2017). [gCoda: Conditional Dependence Network Inference for Compositional Data](https://pubmed.ncbi.nlm.nih.gov/28489411/). Journal of Computational Biology, 24: 699-708.
1. Wang Z+, **Fang H**+, Tang NLS and Deng M (2017). [VCNet: vector-based gene co-expression network construction and its application to RNA-seq data](https://pubmed.ncbi.nlm.nih.gov/28334366/). Bioinformatics, 33: 2173-2181. (+Co-first authors)
1. Wu C, Zhao H, **Fang H** and Deng M (2017). [Graphical model selection with latent variables](https://projecteuclid.org/euclid.ejs/1507255612). Electronic Journal of Statistics, 11: 3485-3521.
1. **Fang H**, Huang C, Zhao H and Deng M (2015). [CCLasso: correlation inference for compositional data through Lasso](https://pubmed.ncbi.nlm.nih.gov/26048598/). Bioinformatics, 31: 3172-3180.
1. Chen J, Peng Z, Zhang R, Yang X, Tan B, **Fang H**, Liu C, Shi M, Ye Z, Zhang Y, Deng, M, Zhang X and Li C (2014). [RNA editome in rhesus macaque shaped by purifying selection](https://pubmed.ncbi.nlm.nih.gov/24722121/). PLoS Genetics, 10: e1004274.
